autoware_adapi_v1_msgs/vehicle/msg/TurnIndicator.msg

```diff
@@ -0,0 +1,6 @@
+# constants
+uint8 DISABLE = 1
```

Comment:
Could you please explain the following items?
- Why isn't it `NONE`?
- Why is it starting from 1, not 0?

We need to create guidelines by clarifying design policies, in order to make the styles consistent throughout the entire Autoware.

Reply:
@kenji-miyake 
Because I refer to this message, https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_vehicle_msgs/msg/TurnIndicatorsReport.idl

But Ya, I agree with you idea. If the api do not receive any msgs from the vehicle report, we should publish `NONE` instead of `DISABLE`.

There is 2 phrases that can be use for 0, `NONE` or `UNKNOWN` .
Personally I feel that `NONE` will be better for 0 label, how do you think?

Reply:
I see. If it's a command, `enable/disable` sound good to me. If it's a status, a little weird.

> If the api do not receive any msgs from the vehicle report, we should publish NONE instead of DISABLE.

If you want to support this use case, I think it's okay to add both `NONE` and `DISABLED`!

By the way, are you planning to support the Hazard lamp in another message? :thinking:

Reply:
> Why is it starting from 1, not 0?

The API has a policy of not using zero for valid data.
https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/#constants-and-enumeration

Reply:
Thank you. Seeing it documented, it's excellent!

Reply:
> If you want to support this use case, I think it's okay to add both NONE and DISABLED!

Yes, I will support this case and add a `NONE` value

> By the way, are you planning to support the Hazard lamp in another message? 

The hazard light is supported and added in this PR, I will add the `NONE` case as well
https://github.com/autowarefoundation/autoware_adapi_msgs/blob/8ca0be545ae6fff36660ad33aa3ba7e1578208fd/autoware_adapi_v1_msgs/vehicle/msg/HazardLight.msg

Reply:
added NONE
[2630db2](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/24/commits/2630db2576ddbee83b26c8d3e8f3cd24cd8e6b1a)

Reply:
By the way, should it be `UNKNOWN` to be consistent with other messages?

Reply:
I think UNKNOWN is better for consistency.

Reply:
sorry for the long wait
I fixed in 48a7263